fossunited/scheduled_tasks.py

```
@@ -27,3 +27,43 @@
+            <p>If you wish to keep this job open, please repost or update the job post.</p>
+            <p>Regards,<br>FossUnited Team</p>
+            """
+        )
+
+        try:
+            jobs.save(ignore_permissions=True)
+            frappe.db.commit()
+        except Exception as e:
+            frappe.log_error(
+                frappe.get_traceback(),
+                f"Error while Updating status of past jobs as Expired through scheduler- ID:{jobs.name}\nError:{str(e)}",
```

Ruff (0.12.2)

Line too long (121 > 99)

(E501)